mass_mailing_partner/models/mail_mass_mailing_contact.py

```diff
@@ -45,6 +45,10 @@ def create(self, vals):
             record._set_partner()
         record._onchange_partner_mass_mailing_partner()
         new_vals = record._convert_to_write(record._cache)
+        new_vals.update(
+            subscription_list_ids=vals.get('subscription_list_ids', False),
```

Comment:
This is breaking test of previous modules on the repository as `mail_tracking_mass_mailing` one of the test asume the the contact is not subscribed before. So whether fix this module would require reduce the improvements or we should refactor first the other module. I guess the PSC of this repository should decide 

cc @chienandalu @ernestotejeda @pedrobaeza @rafaelbn

Reply:
The problem was with the code in module `mail_tracking`, that was calling method `write` inside other `write` call. Fixed in 007d8ada83dec2e1e082064b7a256907baba4817.